music_player_app/lib/common/models/audio_model.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:just_audio/just_audio.dart';
+
+class AudioPlayerModel extends ChangeNotifier {
+  static final audioPlayer = AudioPlayer();
+  bool _isPlaying = false;
```

Comment:
@shraddhashetty-27 You need to use `_isPlaying`  to chaange the play or pause icon

Reply:
Okay.